.github/workflows/integration-test.yml

```diff
@@ -95,13 +121,21 @@ jobs:
       # for more information: http://sealiesoftware.com/blog/archive/2017/6/5/Objective-C_and_fork_in_macOS_1013.html)
       OBJC_DISABLE_INITIALIZE_FORK_SAFETY: 'YES'
 
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_US_EAST_1_ENV_ACCESS_KEY_ID }}
```

Comment:
Not necessary for this PR, but maybe we can optimize these regions to be as close as possible to our GH runner regions (if those are even fixed/we can find out about them)?

Reply:
I did some digging. Apparently it's not possible to predict where the runners are located and they're basically all over the place (EU, US, Asia).

Reply:
From an older issue: ```
GitHub hosts runners in the following Azure regions:

East US (eastus)
East US 2 (eastus2)
West US 2 (westus2)
Central US (centralus)
South Central US (southcentralus)```

So definitely US based is better. Which means I should switch the GCP region to something in the US as well.

Reply:
done